api/ssh.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import "fmt"
+
+// SSH is used to return a client to invoke operations on SSH backend.
+type SSH struct {
+	c *Client
+}
+
+// SSH is used to return the client for logical-backend API calls.
+func (c *Client) SSH() *SSH {
```

Comment:
I have made SSH() take path as param and using it in all the cases. By default, 'ssh' will be passed as param.